src/Shared/Microsoft.DotNet.Scaffolding.Shared/ConsoleLogger.cs

```diff
@@ -8,8 +8,11 @@ internal class ConsoleLogger : IConsoleLogger
         private readonly bool _jsonOutput;
         private readonly bool _silent;
 
-        public ConsoleLogger(bool jsonOutput = false, bool silent = false)
+        private string CommandName { get; }
```

Comment:
also can set the default value to be an empty string, check comments below for context.